springboot-react-fullstack/src/frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
all files under the folder frontend are generated by `npx create-react-app  frontend` command.